app/app.py

```diff
@@ -66,7 +66,11 @@ def init_handlers(self):
 
     def init_signal_handlers(self):
         signal.signal(signal.SIGINT, self.interrupt_handler)
-        signal.signal(signal.SIGQUIT, self.interrupt_handler)
+        # check for windows signal not existing
+        if getattr(signal, 'SIGQUIT', None) is not None:
```

Comment:
Sure yeah I'll do that.  Quick question though:  when I make the change, will it update in the pull request, or do I have to cancel and open a new pull request?